src/libs/elektra/keyname.c

```diff
@@ -1552,17 +1618,17 @@ ssize_t keyAddBaseName (Key * key, const char * baseName)
 }
 
 /**
- * Sets @c baseName as the new basename for @c key.
+ * Sets @p baseName as the new basename for @p key.
  *
- * Only the baseName will be affected and no other part of the key.
+ * Only the basename will be affected and no other part of the Key.
```

Comment:
Sounds weird to me, here are two other variants.
Simple variant:
```suggestion
 * Only the basename of the Key will be affected.
```

Reply:
Second variant
```suggestion
 * Only the basename, and no other part of the Key, will be affected.
```

Reply:
First variant seems good to me - Occam's razor and stuff :wink: